### ISSUE TITLE
Dropdown scrollbar problem in IE10+

### ISSUE THREAD

**Issue by cbier** [Formstone/Formstone#144]
Hey @benplum! Hope you're doing good.

In IE10+ I am having an issue where you cannot click and drag the scrollbar. It works in IE 9 and lower. I haven't tried Edge yet.

The issue appears on the formstone.it demo page too.

**Comment by cbier** [comment]
Please let me know if you have any ideas. Thanks man!

**Comment by rzubov** [comment]
Hi there,
This can be fixed by replacing:
e.pageY => e.originalEvent.pageY
e.pageX => e.originalEvent.pageX

https://github.com/Formstone/Formstone/blob/master/src/js/scrollbar.js#L389-L391

**Comment by cbier** [comment]
@rzubov thanks so much for replying! Unfortunately this does not resolve the issue so I'm still looking into it. Can you tell me more about the solution? Why does adding `originalEvent` solve the issue?

**Comment by cbier** [comment]
After digging deeper I think I see the issue. In `onPan()`, `e.deltaY` is coming out as an incorrect negative number in IE10 and IE11 even when clicking and dragging the scrollbar just a couple of pixels:

IE10: `-1198`
IE11: `-928.010009765625`

In Chrome it behaves as you would expect, and starts out at `1` as you scroll down and increments correctly.

Since the values of `e.deltaY` are incorrect, `positionContent()` does not apply the position because the position is out of the `handleBounds`. This seems to be the root cause of why IE is not scrolling correctly.

**Comment by cbier** [comment]
In IE10 and higher `touches` is defined in the `data` object. In IE9 and lower it is not defined.

In `onPointerMove()` in touch.js, if `touches` is defined, then `newX` and `newY` are set to the respective touch data rather than the regular event data.

`newX` and `newY` are then used to determine `deltaX` and `deltaY`. Being that this is a difference between IE10 and higher and the rest of the IE versions, this could be the root cause of the issue.

Still trying to debug this... any input is appreciated

**Comment by cbier** [comment]
The values in `data.touches[0]` are very low numbers. Therefore, when `data.startX` and `data.startY` are subtracted from those values in `onPointerMove()`, we end up getting those negative numbers (see two comments up).

If I change

```
newX = (touch) ? touch.pageX : e.pageX,
newY = (touch) ? touch.pageY : e.pageY,
```

to

```
newX = e.pageX,
newY = e.pageY,
```

clicking and dragging the scrollbar works in IE11. In IE10, it works as well but periodically glitches by jumping to the top of the dropdown options for a split second.

Of course you put those conditionals in there for a reason. I just don't know why, and I don't know where to go from here.

Thanks again @benplum for the great plugin. If you have some time, I'd greatly appreciate your input.

**Comment by benplum** [comment]
Thanks for spending so much time investigating the issue. For IE10, are you on a machine that supports touch? I'm seeing the problem in IE11, but not 10, and am curious what the difference in our set ups could be. 

The touch values are used to normalize the different events across all platforms, since IE introduced a 'pointer events' model that differs from the webkit/firefox 'touch events' model.

It looks like this issue does not exist in the 0.9.0 branch: http://beta.formstone.it/components/scrollbar/demo/

The plan is to have this new release out soon, but I'll try to investigate why it's broken in 0.8.x.

**Comment by benplum** [comment]
I just found the problem and will be pushing a fix shortly.

**Comment by benplum** [comment]
Fixed in 0.8.40.
